waku/common/envelope.go

```diff
@@ -15,7 +15,7 @@ type Envelope struct {
 	hash  MessageHash
 }
 
-type tmpWakuMessageJson struct {
+type nwakuMessage struct {
```

Comment:
I wouldn't mention nwaku in the type. The suggestion applies elsewhere
```suggestion
type wakuMessage struct {
```